### DISCUSSION TITLE
Proposal: Move proposals to GitHub discussions

### DISCUSSION THREAD

**Discussion by kuizeo** [Pokemon-Standards-Consortium/standards#19] [General]
GitHub Discussions are much better suited for proposals, with boards to hierarchically sort posts (e.x. an "Announcements" board, a "New Standards" board, a "Miscellaneous" board, etc). Issues can then be used for revisions to existing proposals, so we can have a "PSC 001" tag, a "PSC 002" tag, etc (like we already have, except the Proposal tag can be removed).

**Comment by immewnity** [comment]
GitHub Discussions didn't exist when this project began, but I completely agree that this makes sense! Will enable Discussions and start building it out.

**Comment by kuizeo** [comment]
Great! (By the way - GitHub has a feature to convert issues to discussions.)
